README.md

```diff
@@ -28,6 +28,25 @@ foo.render('blah').body
 //=> 'blah'
 ```
 
+## Development
+
+Please feel free to submit pull requests for any feature requests, bug fixes etc.
+
+To test your code locally:
+
+- `yarn install` - install dependencies
+- `yarn test` - runs tests
```

Comment:
While I understand the benefits out of yarn, much of what made it unique is now available in npm itself. Many users install node, but not yarn. Could we use npm as the default in the documentation? Those familiar with yarn will use it instead anyway.